web/src/services/api/layersApi/index.ts

```diff
@@ -72,7 +72,7 @@ export default () => {
 
         return { status: "error", errors };
       }
-      setNotification({ type: "success", text: t("Successfully updated a the layer!") });
+      setNotification({ type: "success", text: t("Successfully updated a layer!") });
```

Comment:
```suggestion
      setNotification({ type: "success", text: t("Successfully updated the layer!") });
```
One small issue, since this is for update one certain layer, is it better to keep `the` instead of `a`?